op-scim-bridge/schema.yaml

```diff
@@ -1,32 +1,47 @@
-application_api_version: v1beta1
+x-google-marketplace:
+  schemaVersion: v2
+  applicationApiVersion: v1beta1
+  publishedVersion: "1.6.2"
+  publishedVersionMetadata:
+    releaseNote: >-
+      Please see the SCIM bridge release notes.
+    releaseTypes:
+      - Feature
+    recommended: true
+
+  images:
+    '':
+      properties:
+        bridge.image.repository:
+          type: REPO_WITH_REGISTRY
+        bridge.image.tag:
+          type: TAG
+    redis: 
+      properties:
+        redis.image.repository:
+          type: REPO_WITH_REGISTRY
+        redis.image.tag:
+          type: TAG
+
+
 properties:
-  APP_INSTANCE_NAME:
+  name:
     type: string
     x-google-marketplace:
       type: NAME
-  NAMESPACE:
+  namespace:
     type: string
     x-google-marketplace:
       type: NAMESPACE
-  IMAGE_OP_SCIM:
-    type: string
-    default: $REGISTRY:$TAG
-    x-google-marketplace:
-      type: IMAGE
-  IMAGE_REDIS:
-    type: string
-    default: $REGISTRY/redis:$TAG
-    x-google-marketplace:
-      type: IMAGE
-  OP_ACCOUNT_DOMAIN:
+  accountdomain:
```

Comment:
What are your thoughts on camel casing these variables for readability? Do you know of any kubernetes values naming standards?

Curious to hear more re the Property naming changes.

Reply:
The overall idea was to bring the variable names more in-line with what is presented in Google's Marketplace examples, as seen here: https://github.com/GoogleCloudPlatform/marketplace-k8s-app-example/blob/master/wordpress/schema.yaml

You're correct, however, in that `accountdomain` should be camelcased. I've gone ahead and done that in the latest commit.